netsim/devices/arubacx.py

```diff
@@ -27,6 +28,18 @@ def device_quirks(self, node: Box, topology: Box) -> None:
                 return
             node.vrfs[vrf]['ospfidx'] = ospfidx
             ospfidx = ospfidx + 1
+    # Remove OSPF default originate route-type (not supported, yet)
+    if 'ospf' in mods:
```

Comment:
FWIW, you could have used `if node.get('ospf.default.type',None)` -- the get functionality on a box survives hierarchical stuff that's not there, and does not modify the box if you supply the default value.